apps/style/code-studio/petition.scss

```diff
@@ -0,0 +1,116 @@
+.petition_block {
+  width: 970px;
+  display: flex;
+  flex-flow: column wrap;
+  text-align: center;
+  background-color: white;
+  padding: 10px;
+}
+
+.petition_message {
+  margin: 0;
+  font-size: 21px;
+  line-height: 28px;
+  font-family: 'Gotham 5r', sans-serif;
+  color: #5B6770;
+}
+
+.petition_sign_message {
+  font-size: 18px;
+  line-height: 20px;
+  font-weight: 400;
+  margin: 10px 0 20px 0;
+  font-family: 'Gotham 4r', sans-serif;
+  color: #5B6770;
+}
+
+.petition_form {
+  font-size: 12px;
+  line-height: 18px;
+  color: #dcdcdc;
+  text-align: left;
+  display: flex;
+  flex-flow: row wrap;
+  flex: 1 100%;
+  justify-content: space-around;
+}
+
+input.field, select.field {
+  background-color: #e2e4e3;
+  color: #595959;
+  padding: 5px;
+  border-width: 1px;
+}
+
+input.grey_input {
+  width: 150px;
+}
+
+span.grey_input {
+  width: 160px;
+}
+
+select.dropdown {
+  width: 93px;
+}
+
+span.dropdown {
+  width: 130px;
+}
+
+.dropdown_label {
+  margin: 0 4px 0 0;
+  color: #5B6770;
+  width: 100px;
+  font-size: 14px;
+  vertical-align: middle;
+}
+
+.go_button {
+  min-width: 100px;
+  margin: 0;
+  padding: 4px 12px;
+  height: 30px;
+  font-family: 'Gotham 5r', sans-serif;
+}
+
+.help, a {
+  font-size: 12px;
+  line-height: 18px;
+  margin: 10px 0 0 0;
+  text-decoration: none;
+  color: #5B6770;
+}
+
+a:hover {
```

Comment:
also, this is fine because this file will only be included in this page, and it'd be even more okay if we had actual css modules set up, but heads up that styling without classNames like this could potential affect things in the header for example

Reply:
oh wait actually yeah, this is that component that shows up on the home page, I would make this selector more specific by adding a className

Reply:
without css modules, when we include an scss file, those rules apply to everything on the page, which is one (arguable) downside compared to inline styles

Reply:
Ah, thanks. I'll add the .help class here too just to be safe